client/views/filters.html

```diff
@@ -9,7 +9,12 @@
 						<label class="active" for="filterbytags">Tags</label>
 					</div>
 					<div class="input-field col s12 m3">
+						{{#if currentUser}}
 						<button class="btn">Add tag</button>
+						{{else}}
+						<button class="btn disabled">Add tag</button>
+						<p class="error">Please log in to add filters</p>
```

Comment:
And same here

Reply:
Login is not a link in the already existing messages, but sure